signup/index.php

```diff
@@ -2,118 +2,130 @@
+			<!-- TemplateEndEditable -->
 		</div>
-		<div class="wdn-menu-trigger wdn-content-slide">
-			<label for="wdn_menu_toggle" class="wdn-icon-menu">Menu</label>
-			<?php wdnInclude("/wdn/templates_4.1/includes/share.html"); ?>
+		<!-- TemplateBeginEditable name="herogroup2" -->
+		<div class="dcf-hero-group-2">
 		</div>
-		<main id="wdn_content_wrapper" role="main" class="wdn-content-slide" tabindex="-1">
-			<div id="maincontent" class="wdn-main">
-				<div id="pagetitle">
-					<!-- TemplateBeginEditable name="pagetitle" -->
-					<h1>Sign Up for a VITA Appointment</h1>
-					<!-- TemplateEndEditable -->
-				</div>
-				<!-- TemplateBeginEditable name="maincontentarea" -->
-				<sign-up class="wdn-band" id="signupApp"></sign-up>
-				<!-- TemplateEndEditable -->
-			</div>
-		</main>
-		<footer id="footer" role="contentinfo" class="wdn-content-slide">
-			<div id="wdn_optional_footer" class="wdn-band wdn-footer-optional">
-				<div class="wdn-inner-wrapper">
-					<!-- TemplateBeginEditable name="optionalfooter" -->
-					<!-- TemplateEndEditable -->
-				</div>
-			</div>
-			<div id="wdn_local_footer" class="wdn-band wdn-footer-local">
-				<div class="wdn-inner-wrapper">
-					<!-- TemplateBeginEditable name="contactinfo" -->
-					<?php include "$root/sharedcode/localFooter.html"; ?>
-					<!-- TemplateEndEditable -->
-					<!-- TemplateBeginEditable name="leftcollinks" -->
-					<!-- TemplateEndEditable -->
-				</div>
-			</div>
-			<div id="wdn_global_footer" class="wdn-band wdn-footer-global">
-				<div class="wdn-inner-wrapper">
-					<?php wdnInclude("/wdn/templates_4.1/includes/globalfooter.html"); ?>
-				</div>
-			</div>
-		</footer>
-		<?php wdnInclude("/wdn/templates_4.1/includes/noscript.html"); ?>
+		<!-- TemplateEndEditable -->
 	</div>
-	<?php wdnInclude("/wdn/templates_4.1/includes/body_scripts.html"); ?>
-	<?php require_once "$root/server/global_includes.php"; ?>
-	<script src="/dist/signup/signup.js"></script>
+	<div class="dcf-main-content dcf-wrapper">
+		<!-- TemplateBeginEditable name="maincontentarea" -->
+		<sign-up id="signupApp"></sign-up>
+		<!-- TemplateEndEditable -->
+	</div>
+</main>
+<footer class="dcf-footer" id="dcf-footer" role="contentinfo">
+	<!-- TemplateBeginEditable name="optionalfooter" -->
+	<!-- TemplateEndEditable -->
+		<?php wdnInclude("/wdn/templates_5.0/includes/global/footer-global-1.html"); ?>
+	<!-- TemplateBeginEditable name="contactinfo" -->
+		<?php include "$root/sharedcode/localFooter.html"; ?>
+	<!-- TemplateEndEditable -->
+		<?php wdnInclude("/wdn/templates_5.0/includes/global/footer-global-2.html"); ?>
+</footer>
+<?php wdnInclude("/wdn/templates_5.0/includes/global/noscript.html"); ?>
+<?php wdnInclude("/wdn/templates_5.0/includes/global/js-body.html"); ?>
+<!-- TemplateBeginEditable name="jsbody" -->
+<?php require_once "$root/server/global_includes.php"; ?>
+<script src="/dist/signup/signup.js"></script>
+<!-- <script src="/dist/assets/js/utilities/button.js"></script> -->
```

Comment:
What's the reason this is commented out? Should we remove this or uncomment it?

Reply:
Ah I see below in `signup.js` this is included. I'm guessing we can remove this part then. Additionally, this is probably the reason that the questionairre buttons don't work--it's not included in the `questionairre.js`?